FWCore/Services/plugins/Timing.cc

```diff
@@ -220,8 +230,8 @@ namespace edm {
           configuredInTopLevelProcess_{false},
           nSubProcesses_{0} {
       iRegistry.watchPreBeginJob(this, &Timing::preBeginJob);
-      iRegistry.watchPostBeginJob(this, &Timing::postBeginJob);
-      iRegistry.preEndJobSignal_.connect([this]() {
+      iRegistry.watchPostBeginProcessBlock(this, &Timing::postBeginProcessBlock);
```

Comment:
@wddgit Could you confirm that the `postBeginProcessBlockSignal_` is issued only for the beginning of the ProcessBlock of the current process, and not for the ProcessBlocks stored in the input file? (I believe this is the case, but want to be sure)

Reply:
I'm more after that the ProcessBlocks of the earlier processes (from the input file) do not cause this signal to be issued.

Reply:
Yes, I confirm that is correct. There is a separate signal for the input process block transitions.

Reply:
I have not read the whole PR carefully and I am not sure if this is relevant to this PR, but possibly this is worth mentioning. There are some corner cases where there can be more than one begin ProcessBlock transition in the same process. From the TWIKI:

"Normally, there is one beginProcessBlock and one endProcessBlock transition in a process. There can be more than one of each in the exceptional cases of EDLooper, SubProcess, and single OutputModules that are configured to write multiple files."